tests/assets/deployment.template_without_schema.json

```diff
@@ -0,0 +1,89 @@
+{
```

Comment:
Minor naming comment: this file should be better named `deployment.template_without_schema_template.json`

Reply:
updated code

Reply:
updated code